src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(Color color, int number) {
+        this.color = String.valueOf(color);
```

Comment:
Please, [do not use String.valueOf](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants) with enum values 
In this case its better to have constructor `public Ball(String color, int number)` and method `getRandomColor()` that returns String

Reply:
done